mkgdaldist.sh

```diff
@@ -141,7 +141,7 @@ CWD=${PWD}
 #
 echo "* Generating man pages..."
 
-(cd doc; make man)
+(cd doc; SPHINXOPTS='--keep-going -j auto' make man)
```

Comment:
This makes it so the warnings aren't turned into errors and it completes successfully even without the python bindings.